x-pack/plugins/infra/public/components/asset_details/tabs/processes/processes_table.tsx

```diff
@@ -205,6 +213,28 @@ const LoadingPlaceholder = () => {
   );
 };
 
+interface ProcessesTableErrorProps {
+  error: string;
+}
+
+const ProcessesTableError = ({ error }: ProcessesTableErrorProps) => (
+  <EuiTableRow>
+    <EuiTableRowCell
+      data-test-subj="infraAssetDetailsProcessesSearchInputError"
+      style={{
+        paddingTop: '8px',
+        paddingBottom: '8px',
+      }}
+      align="center"
+      colSpan={columns.length + 1}
+      mobileOptions={{ width: '100%' }}
+      textOnly={true}
+    >
+      <EuiIcon type="minusInCircle" color="danger" /> {error}
+    </EuiTableRowCell>
+  </EuiTableRow>
+);
```

Comment:
Nice! It might be even better if we could use a hook called `useEuiTheme` to get those padding sizes:

```suggestion
const ProcessesTableError = ({ error }: ProcessesTableErrorProps) => {
  const { euiTheme } = useEuiTheme();
  return (
    <EuiTableRow>
      <EuiTableRowCell
        data-test-subj="infraAssetDetailsProcessesSearchInputError"
        style={{
          paddingTop: `${euiTheme.size.s}`,
          paddingBottom: `${euiTheme.size.s}`,
        }}
        align="center"
        colSpan={columns.length + 1}
        mobileOptions={{ width: '100%' }}
        textOnly
      >
        <EuiIcon type="minusInCircle" color="danger" /> {error}
      </EuiTableRowCell>
    </EuiTableRow>
  );
};
```